opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -682,7 +709,7 @@ namespace cpgrid
                     cells[cellcount].setValue(fnc[1], false);
                     ++cellcount;
                 }
-
+                /*
```

Comment:
Even if the current ewoms code would not need `Intersection::indexInSide()`, we are losing information here because NNCs created by pinch are not treated as regular intersections anymore.

Reply:
I think that is more precise though: the created faces are not normal K-faces. They will still be intersections though.